vhdbuilder/packer/install-dependencies.sh

```diff
@@ -242,8 +265,8 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
 EOF
 
 echo "${CONTAINER_RUNTIME} images pre-pulled:" >> ${VHD_LOGS_FILEPATH}
-
-
+stop_watch $capture_time "Pull NVIDIA Image, Start installBcc subshell" false
```

Comment:
nit: "Pull NVIDIA driver image (mcr)" - minor change to note this is the image from MCR/hosted by Azure. There's a separate Nvidia driver container which will be used in the future.